### PR TITLE
[core] Remove ngraph/op, ngraph/opsets API

### PR DESCRIPTION
**Pull request by praasz**
### Details:
 - Remove `ngraph/op` API
 - Remove 'ngraph/opsets` API
 - These are not remove as used by `opnevino contrib` and `vpux` plugin (will be removed separatly:
   - ngraph/op/split.hpp
   - ngraph/op/concat.hpp
   - ngraph/op/util/op_types.hpp
   - ngraph/op/util/attr_types.hpp
   - ngraph/opsets/opset3.hpp
   - ngraph/op/parameter.hpp
   - ngraph/op/divide.hpp

### Tickets:
 - [CVS-129009](https://jira.devtools.intel.com/browse/CVS-129009)

### CODE REVIEW

File: src/core/template_extension/old/op.hpp
Comment (ilya-lavrenov) [inline src/core/template_extension/old/op.hpp:1]: looks like we can remove old extension as well.

File: src/plugins/intel_cpu/tests/functional/subgraph_tests/src/concat_reorder_inplace.cpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <ngraph/opsets/opset8.hpp>
+// #include "openvino/opsets/opset8.hpp"
```
Comment (ilya-lavrenov) [inline src/plugins/intel_cpu/tests/functional/subgraph_tests/src/concat_reorder_inplace.cpp:5]: remove?

File: src/frontends/onnx/frontend/src/core/value_info.hpp [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 #include "core/tensor.hpp"
 #include "default_opset.hpp"
-#include "ngraph/op/constant.hpp"
+//
```
Comment (ilya-lavrenov) [inline src/frontends/onnx/frontend/src/core/value_info.hpp:11]: remove?

File: src/frontends/onnx/frontend/include/onnx_import/core/node.hpp [hunk 3]
```diff
@@ -20,7 +20,7 @@
 #include "ngraph/deprecated.hpp"
 #include "ngraph/except.hpp"
 #include "ngraph/node.hpp"
-#include "ngraph/op/constant.hpp"
+//
```
Comment (ilya-lavrenov) [inline src/frontends/onnx/frontend/include/onnx_import/core/node.hpp:23]: remove?

**Review by vurusovs — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

File: src/core/template_extension/old/op.hpp
Comment (praasz) [reply src/core/template_extension/old/op.hpp:1]: Removed in #22146

**Review by t-jankowski — APPROVED** [review]

**Review by p-durandin — APPROVED** [review]

**Pull request merged**